pgx/src/pgx.ml

```diff
@@ -453,14 +453,14 @@ module Make (Thread : IO) = struct
   open Thread
 
   type conn =
-    { ichan: in_channel sexp_opaque (* In_channel wrapping socket. *)
-    ; chan: out_channel sexp_opaque (* Out_channel wrapping socket. *)
```

Comment:
Oh I see. What's wrong with sexp_opaque?

Reply:
`sexp_opaque` was moved to be `[@sexp.opaque]` in the newer versions of ppx_sexp_conv. 
I didn't think i had way to have this work with both 0.11 and 0.13 without writing handwritten converters for the opaque items